Self-service scoping: documentation, validation, metadata. (#111)
Software-Improvement-Group/sigridci: docs/azure-devops.md

- You have a [Sigrid](https://sigrid-says.com) user account. 
- You have created an [authentication token for using Sigrid CI](authentication-tokens.md).
- [Python 3.7 or higher](https://www.python.org) needs to be available in the CI environment. The client script for Sigrid CI is based on Python.

> - The examples assume [Git](https://git-scm.com) is available in the CI environment.

Reviewer: ```suggestion
- The examples assume [Git](https://git-scm.com) is available on your Azure DevOps environment.
```
